CMakeLists.txt

```diff
@@ -69,6 +69,13 @@ option(USE_EXTERNAL_SINGLEAPPLICATION "Use external QtSingleApplication library"
 option(USE_LAUNCHER_ABSOLUTE_PATH "Use absolute path for the desktop launcher" ON)
 option(USE_WAYLAND_CLIPBOARD "USE KF Gui Wayland Clipboard" OFF)
 option(DISABLE_UPDATE_CHECKER "Disable check for updates" OFF)
+option(USE_PLUGIN_MANAGER "Activate the Plugin Manager" ON)
+option(USE_WAYLAND_GRIM "Activate the Wayland GRIM screenshot adapter" OFF)
```

Comment:
I think this is irrelevant to this PR

Reply:
The first flag is related to the plugin manager, the second one is actually a fix for another function in the CMakeList.txt.

Reply:
If you want to remove line 73, that's fine.

Reply:
I meant the line 73 which is about Grim. That is out of the scope of this PR as far as I understand

Reply:
done